build/build-buildpacks-images.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -e
+
+declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
+source $REPO_DIR/build/__variables.sh
+
+# Build base image for builder
+cd "$BUILD_IMAGES_BUILD_CONTEXT_DIR"
+docker build -f "$BUILDER_BASE_IMAGE_DOCKERFILE" -t $DOCKER_BUILDER_BASE_IMAGE_REPO:latest .
+
+cd /tmp
+
+$REPO_DIR/images/buildpack-builder/install-pack.sh
```

Comment:
Based on this script content, this will only run on non-windows platforms?

Reply:
Correct. In any case, `pack create-builder` is unsupported on Windows